doc/user/content/administration/billing.md

```diff
@@ -0,0 +1,113 @@
+---
+title: "Usage & billing"
+description: "Understand the billing model of Materialize, and learn best practices for cost control."
+menu:
+  main:
+    parent: "manage"
+    weight: 50
+---
+
+{{< note >}}
+Access to usage and billing requires **administrator** privileges.
+{{</ note >}}
+
+Materialize determines billing based on your storage and compute usage.
+
+## Invoice
```

Comment:
Maybe kick this to the bottom? It feels like the right ordering here will be: Compute, Storage, then Invoices!